pkg/bzz/address.go

```diff
@@ -28,33 +27,30 @@ var ErrInvalidAddress = errors.New("invalid address")
 // It consists of a peers underlay (physical) address, overlay (topology) address and signature.
 // Signature is used to verify the `Overlay/Underlay` pair, as it is based on `underlay|networkID`, signed with the public key of Overlay address
 type Address struct {
-	Underlay        ma.Multiaddr
+	Underlay        []ma.Multiaddr
 	Overlay         swarm.Address
 	Signature       []byte
 	Nonce           []byte
 	EthereumAddress []byte
 }
 
 type addressJSON struct {
-	Overlay   string `json:"overlay"`
-	Underlay  string `json:"underlay"`
```

Comment:
As we discussed at some point, we should keep the Underlay field and have the underlay of p2p connection there, for example, as that is the case with the current code and it makes the api backward compatible. We can deprecate the field gradually.